swift/swift.go

```diff
@@ -453,16 +455,54 @@ func (o *FsObjectSwift) Open() (in io.ReadCloser, err error) {
 	return
 }
 
+func Min(x, y int64) int64 {
+    if x < y {
+        return x
+    }
+    return y
+}
+
 // Update the object with the contents of the io.Reader, modTime and size
 //
 // The new object may have been created if an error is returned
 func (o *FsObjectSwift) Update(in io.Reader, modTime time.Time, size int64) error {
+	segmentSize, err := strconv.ParseInt(fs.ConfigFile.MustValue(o.swift.Name(), "segment-size"), 10, 64)
```

Comment:
Why isn't this an integer to begin with?
The name "MustValue" indicates that it will probably panic if the value isn't found - I am pretty sure we don't want that.